tsconfig.json

```diff
@@ -0,0 +1,27 @@
+{
+  "compilerOptions": {
+    "allowSyntheticDefaultImports": true,
+    "baseUrl": "./",
+    "outDir": "./dist/",
+    "declaration": true,
+    "esModuleInterop": true,
+    "jsx": "react",
+    "lib": ["es6", "es2017.object"],
+    "moduleResolution": "Node",
+    "paths": {
+      "@utils/*": ["src/utils/*"]
+    },
+    "strict": true,
+    "target": "esnext"
+  },
+  "exclude": [
+    "node_modules",
+    "dist",
+    "babel.config.js",
+    "metro.config.js",
+    "jest.config.js",
```

Comment:
Do you really need to explicitly exclude JS files? This seems weird.

Reply:
This was before removing `allowJs`, but I'll remove these excludes.